src/main/java/com/google/sps/data/HiddenGem.java

```diff
@@ -14,33 +14,63 @@
 
 package com.google.sps.data;
 
+import java.net.URL;
+import com.google.maps.model.OpeningHours;
+
 /** Hidden Gems are restaurants or cafes in Australia which have 10-50 reviews and a rating of at least 3.5/5. */
 public final class HiddenGem {
 
-  private final int id;
+  private final String placeId;
```

Comment:
Is this because the place id a string or an int in google's system?

Reply:
Yes it looks like the placeId looks like "ChIJv08I6B-vEmsRSpRhq3Gfbeipvp35VA3k"